src/common/settings.json

```diff
@@ -717,5 +717,12 @@
         "aliases": [
             "user"
         ]
+    },
+    {
+        "name": "zstd_min_string_length",
+        "description": "The (average) length at which to enable ZSTD compression, defaults to ('default_block_size' / 4)",
```

Comment:
It defaults to `4096`. The `default_block_size / 4` is only used for smaller block sizes. Can we update this and turn `zstd_min_string_length` into a `uint64_t`? Then we also don't need the custom implementation of the setting.